k2/csrc/cuda/arch.h

```diff
@@ -0,0 +1,32 @@
+// k2/csrc/cuda/arch.h
+
+// Copyright (c) 2020 Xiaomi Corporation ( authors: Meixu Song )
+
+// See ../../LICENSE for clarification regarding multiple authors
+#ifndef K2_CSRC_CUDA_ARCH_H_
+#define K2_CSRC_CUDA_ARCH_H_
+
+namespace k2 {
```

Comment:
Does it make sense to use a namespace here?

Reply:
can remove arch.h...